src/mono/wasm/runtime/logging.ts

```diff
@@ -131,8 +131,15 @@ export function setup_proxy_console(id: string, console: Console, origin: string
                     }
                 }
 
-                if (typeof payload === "string" && id !== "main")
-                    payload = `[${id}] ${payload}`;
+                const now = new Date().toISOString();
+                if (typeof payload === "string")
+                    if (payload[0] == "[")
+                        if (id !== "main")
+                            payload = `[${id}][${now}] ${payload}`;
+                        else
+                            payload = `[${now}] ${payload}`;
+                    else if (id !== "main")
+                        payload = `[${id}] ${payload}`;
```

Comment:
nit: Don't create `Date` if it's not used

```suggestion
                if (typeof payload === "string") {
                    if (payload[0] == "[") {
                        const now = new Date().toISOString();
                        if (id !== "main") {
                            payload = `[${id}][${now}] ${payload}`;
                        } else {
                            payload = `[${now}] ${payload}`;
                        }
                    } else if (id !== "main") {
                        payload = `[${id}] ${payload}`;
                    }
                }
```